### PR TITLE
Feature/wrap up css styling

### PR DESCRIPTION
**Pull request by lschnell8**
PR Template: 

Is this a fix or a feature?
[] Bug fix
[x] Feature
[] Refactor

Describe the feature - fix - refactor:
Fix non-semantic tags in HTML and finish CSS styling
How was this tested?
[x] open index.html
[] localhost 
[] terminal

Where should the reviewer start - test:
HTML line 39 and CSS review full file for changes and additions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,18 +36,20 @@ <h3 class="words">Filter Starred Ideas</h3>
...
             <img class="star" src="assets/star.svg" alt="star">
             <img class="star hidden" src="assets/star-active.svg" alt="active star">
             <img class="delete" src="assets/delete.svg" alt="delete">
             <!-- <img class="delete hidden" src="assets/delete-active.svg" alt="active delete"> HOVER STATE TOGGLE -->
-          </header>
-          <h4>Idea title</h4>
-          <p>Idea body</p>
-          <footer>    <!--change tag type in html and css-->
-            <img src="assets/comment.svg" alt="add comment">
+          </section>
```
Comment (ninabrissey) [inline index.html:44]: Siginificant changes here to adjust the layout of the <p> tags within the <article>. It looks good. Let's keep an eye on funtionality.

File: styles.css [hunk 2]
```diff
@@ -114,27 +114,46 @@ article {
   width: 30%;
   background-color: #FFFFFF;
   border: 2px solid #1F1F3D;
+  margin: 2.5% 0 0 2.5%;
+  font-size: 1.75em;
```
Comment (ninabrissey) [inline styles.css:118]: Would like to look at ems. Just for reference.

File: styles.css [hunk 3]
```diff
@@ -148,25 +167,29 @@ footer {
 .aside-btn {
   background-color: #5356a4;
   color: #e9e9f3;
-  margin-left: 15%;
-  margin-right: 15%;
+  letter-spacing: 1px;
+  margin: 3.5% 5% 9% 5%;
   cursor: pointer;
```
Comment (csmucker83) [inline styles.css:172]: Add cursor/pointer to all buttons

**Review by csmucker83 — APPROVED** [review]
Tweaked and perfected the css file to begin working on the js files.

**Pull request merged**